google/ggrc-core
# GGRC-6676 "Uncaught TypeError: Cannot read property 'split' of null" error occurs if open Edit modal for person without Profile

# Issue description

"Uncaught TypeError: Cannot read property 'split' of null" error occurs if open Edit modal for person without Profile

# Steps to test the changes

1. Log as admin user to ggrc-test
2. Open Administration page > People tab
3. Select user, e.g. Albert Shin (albert@google.com)
4. Expand info panel > 3 dot's> Edit Person
Actual Result: "Uncaught TypeError: Cannot read property 'split' of null" error occurs if open Edit modal for person.
Expected Result: no errors should be shown. Edit modal should be displayed

# Solution description

Added migration to create missing PersonProfiles
Cleaned up codebase

# Sanity checklist




# PR Review checklist
